dmscripts/notify_buyers_when_requirements_close.py

```diff
@@ -17,6 +17,7 @@
 def notify_users(email_api_key, brief):
     logger.info("Notifying users about brief ID: {brief_id} - '{brief_title}'",
                 extra={'brief_title': brief['title'], 'brief_id': brief['id']})
+    email_client = DMMandrillClient(email_api_key, logger=logger)
```

Comment:
We could easily port this email to Notify as well (in fact I thought we already had done a while back!)